mlir/include/mlir/Dialect/Rock/IR/RockGemmWrapperInterface.td

```diff
@@ -129,9 +129,32 @@ def RockGemmWrapperInterface : OpInterface<"RockGemmWrapperInterface"> {
         /*defaultImplementation=*/[{
           $_op->setAttr($_op.getParamsAttrName(), params);
         }]
+      >,
+    InterfaceMethod<
+        /*desc=*/[{
+          Get Optional blocksize
+        }],
+        /*retType=*/"Optional<uint32_t>",
+        /*methodName=*/"getDerivedBlockSize",
+        /*args=*/(ins),
+        /*methodBody=*/"",
+        /*defaultImplementation=*/""
+        }]
+      >,
+    InterfaceMethod<
+        /*desc=*/[{
+          Get blocksize if it is non-Xdlops and given in the gemmParam
```

Comment:
Wrong semantics, wrong documentation.

`Optional<uint32_t> getBlockSize()` is the signature
The docs are
```
Return the block size this operation will execute with if it has been selected or None otherwise.
Prioritizes the block size defined in tuning parameters
```